picard/ui/options/interface.py

```diff
@@ -36,7 +37,58 @@ class InterfaceOptionsPage(OptionsPage):
     PARENT = "advanced"
     SORT_ORDER = 40
     ACTIVE = True
-
+    SEPARATOR = u'—'*5
+    TOOLBAR_BUTTONS = {
+        'add_directory_action': {
+            'label': N_(u'Add Folder'),
+            'icon': 'folder'
+        },
+       'add_files_action': {
+            'label': N_(u'Add Files'),
+            'icon': 'document-open'
+        },
+       'cluster_action': {
+            'label': N_(u'Cluster'),
+            'icon': 'picard-cluster'
+        },
+       'autotag_action': {
+            'label': N_(u'Lookup'),
+            'icon': 'picard-auto-tag'
+        },
+       'analyze_action': {
+            'label': N_(u'Scan'),
+            'icon': 'picard-analyze'
+        },
+       'browser_lookup_action': {
+            'label': N_(u'Lookup in Browser'),
+            'icon': 'lookup-musicbrainz'
+        },
+       'save_action': {
+            'label': N_(u'Save'),
+            'icon': 'document-save'
+        },
+       'view_info_action': {
+            'label': N_(u'Info'),
+            'icon': 'picard-edit-tags'
+        },
+       'remove_action': {
+            'label': N_(u'Remove'),
+            'icon': 'list-remove'
+        },
+       'submit_acoustid_action': {
+            'label': N_(u'Submit AcoustIDs'),
+            'icon': 'acoustid-fingerprinter'
+        },
+       'play_file_action': {
+            'label': N_(u'Open in Player'),
+            'icon': 'play-music'
+        },
+       'cd_lookup_action': {
+            'label': N_(u'Lookup CD...'),
+            'icon': 'media-optical'
+        },
+    }
+    ACTION_NAMES = set(TOOLBAR_BUTTONS.keys())
     options = [
         config.BoolOption("setting", "toolbar_show_labels", True),
```

Comment:
This option should be moved to Customize Action Toolbar perhaps (as it is the main target)